client/src/js/filters/filters.js

```diff
@@ -1,109 +1,111 @@
+// (function (angular) {
+'use strict';
+
+angular.module('bhima.filters')
+  .filter('boolean', function() {
+    return function (input) {
+      return Boolean(Number(input));
+    };
+  })
+
+  // Potential currency filter providing a specified locale 
+  // Ideally would leverage angular currency filter but this does NOT support
+  // passing through individual locale and doesn't expose utility methods 
+  .filter('intlcurrency', [
```

Comment:
Although it was an essay:
'5. Deprecated the intlcurrency filter - please note this PR request does not replace all instances of the filter. For now it simple returns an error message on use.'

This was mostly just a time issue. I grepped 'intlcurrency' and decided I would make it a separate commit!